openshift/summitbot-deployment-template.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: v1
+kind: Template
+
+labels:
```

Comment:
I think this will not work on this specific level:

have a look at https://github.com/thoth-station/user-api/blob/master/openshift/deployment-template.yaml#L1-L17

Reply:
I should add more metadata to it?

Reply:
I think so :) I love verbose metadata...